exasol_integration_test_docker_environment/test/conftest.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import pytest
+
+from exasol_integration_test_docker_environment.testing import utils
+from exasol_integration_test_docker_environment.testing \
+   .exaslct_test_environment import ExaslctTestEnvironment
+
+
+_logger = logging.getLogger(__name__)
+
+
+class PyTestEnvironment:
```

Comment:
can be implemented as fixture function with yield

```
@pytest.fixture
def itde_cli_test_isolation(request):
    testname = request.node.name
    environment = ExaslctTestEnvironment(self, "itde", clean_images_at_close=False)
    try:
       yield environment
    finally:
       utils.close_environments(spawned, environment)

@contextlib.contextmanager
def database(itde_test_isolation, additional_parameter=None):
       spawned = self.environment.spawn_docker_test_environments(
            name=self.docker_environment_name,
            additional_parameter=additional_parameter,
        )
        try:
            yield spawned
         finally:
            utils.close_environments(self.spawned)

def test(itde_test_isolation):
    with database(itde_test_isolation,additional_parameter = ["--db-os-access", "SSH"])
         ....
```

Reply:
added to branch with required changes to fix imports, etc.